kiwi/builder/disk.py

```diff
@@ -68,6 +73,19 @@
     KiwiVolumeManagerSetupError
 )
 
+StorageMap = TypedDict(
+    'StorageMap', {
+        'system': Optional[Union[FileSystemBase, VolumeManagerBase]],
+        'system_boot': Optional[FileSystemBase],
+        'system_efi': Optional[FileSystemBase],
+        'system_spare': Optional[FileSystemBase],
+        'system_custom_parts': Dict[str, FileSystemBase],
+        'luks_root': Optional[LuksDevice],
+        'raid_root': Optional[RaidDevice],
+        'integrity_root': Optional[IntegrityDevice]
+    }
+)
```

Comment:
The namedtuple like syntax is deprecated as of Python 3.11 and will be removed in python 3.13
```suggestion
class StorageMap(TypedDict):
    system: Optional[Union[FileSystemBase, VolumeManagerBase]],
    system_boot: Optional[FileSystemBase],
    system_efi: Optional[FileSystemBase],
    system_spare: Optional[FileSystemBase],
    system_custom_parts: Dict[str, FileSystemBase],
    luks_root: Optional[LuksDevice],
    raid_root: Optional[RaidDevice],
    integrity_root: Optional[IntegrityDevice]
```

Reply:
I tried this on py3.11 and only got a E999 SyntaxError: invalid syntax

Reply:
That's because I'm stupid and left the trailing commas in there :facepalm:

Reply:
ah yes, thanks :) and done